dev/jenkins_task_validate_image.sh

```diff
@@ -0,0 +1,624 @@
+#!/bin/bash
+#
+# This script configures and deploys Spinnaker on GCP and runs an integration
+# test suite for validation that manages resources on AWS, GCP, and Kubernetes.
+#
+# The following environment variables are expected to be populated:
+#
+# IMAGE_TO_VALIDATE [String] - The name of the image to validate.
+# IMAGE_PROJECT [String] - The GCE project containing the $IMAGE_TO_VALIDATE.
+# If changing this, you will also need to make sure that the $TESTER_SERVICE_ACCOUNT has permission to View (or Edit) the project so that it can download the image.
+# MY_INSTANCE_NAME [String] - The name of the instance to deploy when testing. Must be unique.
```

Comment:
can you rename this to SPINNAKER_INSTANCE_NAME and SPINNAKER_PROJECT and SPINNAKER_ZONE? You may also need a SPINNAKER_SERVICE_ACCOUNT. I dont notice one here because we use the same account for both. But if you are pulling this out as a script then you should decouple that.

Regroup these to collcate the jenkins config from the deplment config from the managed config from the citest config.